src/cloud/live/stage-prod-test.md

```diff
@@ -200,7 +200,9 @@ If you encounter issues, save your reproduction steps, error messages, strange s
 </tr>
 </table>
 
-## Complete load and stress tests {#loadtest}
+## Complete testing
```

Comment:
Seems like this section could be a new topic.

Reply:
Created MAGECLOUD-5168 for adding separate test topic-- there's also devdocs branch that has a draft Performance testing for Magento Cloud -

Reply:
That branch was horribly out of date and it was deleted. However, I saved the work from it.